README.md

```diff
@@ -1,51 +1,110 @@
-# Qᴜᴀʟᴛʀᴀɴ
+<!-- H1 title omitted because our logo acts as the title. -->
+
+<div align="center">
+<img alt="Qualtran logo" width="340px" src="docs/_static/qualtran-logo-black.svg#gh-light-mode-only">
+<img alt="Qualtran logo" width="340px" src="docs/_static/qualtran-logo-white.svg#gh-dark-mode-only">
+<br>
+
+Python package for fault-tolerant quantum algorithms research.
+
+[![Licensed under the Apache 2.0 open-source license](https://img.shields.io/badge/License-Apache%202.0-3c60b1.svg?logo=opensourceinitiative\&logoColor=white\&style=flat-square)](https://github.com/quantumlib/qualtran/blob/main/LICENSE)
+[![Compatible with Python versions 3.10 and higher](https://img.shields.io/badge/Python-3.10+-6828b2.svg?style=flat-square&logo=python&logoColor=white)](https://www.python.org/downloads/)
+[![Qualtran project on PyPI](https://img.shields.io/pypi/v/qualtran.svg?logo=python&logoColor=white&label=PyPI&style=flat-square&color=9d3bb8)](https://pypi.org/project/qualtran)
+
+[Installation](#installation) &ndash;
+[Usage](#usage) &ndash;
+[Documentation](#documentation) &ndash;
+[News](#news) &ndash;
+[Citation](#citation) &ndash;
+[Contact](#contact)
+
+</div>
 
 Qᴜᴀʟᴛʀᴀɴ (quantum algorithms translator) is a set of abstractions for representing quantum
 programs and a library of quantum algorithms expressed in that language to support quantum
 algorithms research.
 
-**Note:** Qualtran is an experimental preview release. We provide no backwards compatibility
```

Comment:
@mpharrigan shall we update this to align better with the stability of versions released 09.2024 onwards